routes.py

```diff
@@ -1,11 +1,14 @@
-from flask import Flask
+from flask import (
+    Flask,
+    render_template,
+)
 
 app = Flask(__name__)
 
 
 @app.route("/")
-def hello_world():
-    return "<h1>Hello, World!</h1>"
+def index():
+    return render_template('index.html', who="World")
```

Comment:
What does the second argument in this line mean? `render_template('path', who=<"What's this?">)`

Reply:
Good question! I was hoping you'd ask that. That basically says that any variables with the name `who` in the template should have the value "World". We could pass anything in there and change who we are saying hello to. It's kind of like a format string, but for the HTML.